src/components/Captures/CaptureGallery.test.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { act, render, screen, within, cleanup } from '@testing-library/react';
+import theme from '../common/theme';
+import { ThemeProvider } from '@material-ui/core/styles';
+import { CapturesContext } from '../../context/CapturesContext';
+import { SpeciesContext } from '../../context/SpeciesContext';
+import { TagsContext } from '../../context/TagsContext';
+import CaptureGallery from './CaptureGallery';
+import {
+  CAPTURES,
+  capturesValues,
+  tagsValues,
+  speciesValues,
+} from '../tests/fixtures';
+
+import * as loglevel from 'loglevel';
+const log = loglevel.getLogger('../tests/verify.test');
+
+jest.setTimeout(7000);
+jest.mock('../../api/treeTrackerApi');
+
+describe('Captures', () => {
+  // mock captures context api methods
+  const getCaptures = () => {
+    log.debug('mock getCaptures:');
```

Comment:
I'm not sure if logging within tests is necessary. I don't think people will look at the test logs if all tests are passing. If they are failing, the failed test names should be enough to determine what's wrong.

Reply:
They've been mainly for debugging the tests, but they do clutter the test view.   Happy to remove.